### PR TITLE
docs: add site-admin capabilities

### PR DESCRIPTION
**Pull request by unknwon**
This PR adds a page to the documentation about site-admin capabilities, i.e. what site-admin can do that regular users can't.

LMK if I missed anything worth documenting!

Fixes sourcegraph/security-issues#61.

### CODE REVIEW

File: doc/_resources/templates/document.html [hunk 1]
```diff
@@ -106,6 +106,7 @@
                     <li class="content-nav-no-hover">
                         <a class="content-nav-section-header" href="/admin">Site admin documentation</a>
                         <ul class="content-nav-section-group">
+                            <li><a href="/admin/capabilities">Site admin capabilities</a></li>
```
Comment (beyang) [inline doc/_resources/templates/document.html]: Can we rename this to "Site admin privileges"?

File: doc/admin/capabilities.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Site administrator capabilities
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
# Site administrator privileges
```

File: doc/admin/capabilities.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+Site administrators have full administrative access to the Sourcegraph instance. In many cases, they also control the deployment environment. Therefore, it is reasonable to have extra capabilities for site administrators.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
Site administrators have full administrative access to the Sourcegraph instance. In many cases, they also control the deployment environment. Special privileges are granted to site-admin users.
```

File: doc/admin/capabilities.md [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+## Access to all repositories
+
+Site administrators are able to access all repositories added to the Sourcegraph instance, and manage settings of individual repository, including repository indexing status, statistics, mirroring information and code intelligence assets.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
Site administrators are able to access all repositories on the Sourcegraph instance and manage the settings of individual repositories.
```

File: doc/admin/capabilities.md [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+## Access to all GraphQL APIs
+
+Site administrators are able to access all GraphQL APIs of the Sourcegraph instance. There are many GraphQL APIs require the client to have the token from a site administrator.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
Site administrators are able to access all GraphQL APIs of the Sourcegraph instance, including special GraphQL APIs that require a site-admin access token.
```

File: doc/admin/capabilities.md [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+
+## Impersonate regular users
+
+Site administrators are able to impersonate regular users in two ways:
+1. View and update settings of individual user.
+2. Initiate GraphQL API calls on behave of a regular user.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
2. Initiate GraphQL API calls on behalf of any other user.
```

File: doc/admin/capabilities.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+Site administrators are able to access all repositories added to the Sourcegraph instance, and manage settings of individual repository, including repository indexing status, statistics, mirroring information and code intelligence assets.
+
+## Access to all GraphQL APIs
+
+Site administrators are able to access all GraphQL APIs of the Sourcegraph instance. There are many GraphQL APIs require the client to have the token from a site administrator.
+
+## Impersonate regular users
+
+Site administrators are able to impersonate regular users in two ways:
+1. View and update settings of individual user.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
1. View and update settings of individual user.
```
```suggestion
1. View and update the settings of any user.
```

File: doc/admin/capabilities.md [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+2. Initiate GraphQL API calls on behave of a regular user.
+
+## Receive site alerts
+
+Site administrators are able to see site alerts in the form of top banner about configuration problems of the Sourcegraph instance.
```
Comment (beyang) [inline doc/admin/capabilities.md]:
```suggestion
Site administrators see update notifications and other site-level alerts (visible as a banner across the top of the screen) that may be invisible to non-admin users.
```

File: doc/admin/index.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
 # Site administration documentation
 
-Site administrators are the admins responsible for deploying, managing, and configuring Sourcegraph for regular users.
+Site administrators are the admins responsible for deploying, managing, and configuring Sourcegraph for regular users. Site administrator also come with [extra capabilities](capabilities.md) compare to regular users.
```
Comment (beyang) [inline doc/admin/index.md]:
```suggestion
Site administrators are the admins responsible for deploying, managing, and configuring Sourcegraph for regular users. They have [special privileges](privileges.md) on the Sourcegraph instance.
```

File: doc/_resources/templates/document.html (hunk 1 above)
Comment (unknwon) [reply doc/_resources/templates/document.html]: Ah, that's much better name!

**Comment by unknwon** [comment]
Thanks @beyang, all comments addressed!

**Comment by unknwon** [comment]
Merging as all sentences are updated by suggestions. Happy to address any post-merge comments!

**Pull request merged**